### PR TITLE
Fix logging in RejectedFilesHandler

### PR DESCRIPTION
**Pull request by rkondratowicz**
add cause

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/blobrouter/services/storage/RejectedFilesHandler.java [hunk 1]
```diff
@@ -88,7 +88,7 @@ private void handleEnvelope(
                 logger.info("Rejected file successfully handled. " + loggingContext);
             }
         } catch (Exception exc) {
-            logger.error("Error handling rejected file. " + loggingContext);
+            logger.error("Error handling rejected file. " + loggingContext, exc);
```
Comment (doncem) [inline src/main/java/uk/gov/hmcts/reform/blobrouter/services/storage/RejectedFilesHandler.java]: just seeing this. in logging statements we should use it's own string interpolation. e.g. `log.level("some string {}", param, exception);`

Comment (rkondratowicz) [reply src/main/java/uk/gov/hmcts/reform/blobrouter/services/storage/RejectedFilesHandler.java]: fixed

Comment (doncem) [reply src/main/java/uk/gov/hmcts/reform/blobrouter/services/storage/RejectedFilesHandler.java]: and the other 2 instances in this method?

Comment (rkondratowicz) [reply src/main/java/uk/gov/hmcts/reform/blobrouter/services/storage/RejectedFilesHandler.java]: I'll change that separately.

**Pull request merged**